src/pages/[genre]/index.tsx

```diff
@@ -61,9 +61,9 @@ export const Home: NextPage<HomeProps> = (props) => {
   const dispatch = useDispatch();
   const route = useRouter();
 
-  const { lazyLoadBIds, genre = 'general' } = props;
+  const { lazyLoadBIds, genre = 'general', branches: _branches } = props;
   const previousGenre = usePrevious(genre);
-  const [branches, setBranches] = useState(props.branches || []);
+  const [branches, setBranches] = useState(_branches || []);
```

Comment:
```suggestion
  const [branches, setBranches] = useState(props.branches || []);
```

`_branches` 는 의미가 불분명한 네이밍으로 느껴집니다.

props.branches 를 초기값으로 사용하는 state 라는게 명확히 보이도록 하는게 좋을 것 같습니다.